fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -2158,6 +2158,7 @@ expression
     | NOT expression                                                                      #logicalNot
     | left=expression operator=(AND|LOGICAL_AND) right=expression                         #logicalBinary
     | left=expression operator=(OR|LOGICAL_OR) right=expression                           #logicalBinary
+    | identifier '=>' expression                                                          #namedArgument
```

Comment:
add it in functioncall is better?

Reply:
or new a special expression `ArgumentExpression` to resolve all function arguments? adding here will contaminate all expressions

Reply:
is `param1 => param2 => value_expr` valid?

Reply:
may use a new rule like `expression ('=>' expression) ?` to decribe the item in table fucntion arguments?

Reply:
introduce namedArgumentList to narrow scop needed.
param1 => param2 => value_expr is invalid.